spec/helpers.rb

```diff
@@ -0,0 +1,9 @@
+module Helpers
+  def create_items
+    @category = Category.create(title: "Cats")
+    @item = @category.items.create(title: "Black Cat Onesie",
+                        description: "The best cat onesie you'll ever find",
+                        price: 19.99,
+                        image: "https://images-na.ssl-images-amazon.com/images/I/41CFHIae7TL._SL500_.jpg")
+  end
```

Comment:
Like this method, might need to refactor my tests with something similar

Reply:
@samuelssnider Yes! I think building on this `helpers.rb` will help us keep our code DRY. 👍 Let me know if you have any questions when you start refactoring. Remember to make your variables an instance variable so you can call them in your own test file.